### PR TITLE
Fix Mac-incompatible use of `head`

### PR DESCRIPTION
**Pull request by matthewlmcclure**
Use `tac` or `tail -r` instead of `head -n -2`

The latter isn't POSIX and isn't available on Mac OS.

### CODE REVIEW

File: bin/vagrant-tramp-ssh [hunk 1]
```diff
@@ -35,6 +47,7 @@ if [[ ! "$name" ]]; then name="default"; fi
 read id dir <<<\
      $(vagrant global-status --machine-readable \
            | head -n-2 | tail -n+8 \
```
Comment (empiricalthought) [inline bin/vagrant-tramp-ssh]: forgot to remove the line calling `head -n-2` here...

**Comment by matthewlmcclure** [comment]
Oops. Feel free to replace this PR with one that's fixed if you want. It
might be a while before I have a chance.

**Comment by eeshugerman** [comment]
It looks like this issue was addressed in https://github.com/dougm/vagrant-tramp/pull/38. Can this be closed?

**Comment by matthewlmcclure** [comment]
Closing as requested

**Pull request closed without merging**